app/components/media/wrapper_component.rb

```diff
@@ -21,7 +21,7 @@ def call # rubocop:disable Metrics/MethodLength,Metrics/AbcSize
                 media_wrapper_index_value: @resource_index,
                 action: 'thumbnail-clicked@window->media-wrapper#toggleVisibility',
                 stanford_only: @file.stanford_only?,
-                location_restricted: @file.location_restricted?,
+                location_restricted: @file.location_restricted? || @file.view_location_restricted?,
```

Comment:
I don't think we want `location_restricted?' we only care about where they can view it here.

Reply:
Thanks @jcoyne. I have made that change.